tests/test_modeling_big_bird.py

```diff
@@ -397,8 +397,9 @@ def prepare_config_and_inputs_for_common(self):
 @require_torch
 class BigBirdModelTest(ModelTesterMixin, unittest.TestCase):
 
-    # head masking is currently not supported for big bird
+    # head masking & pruning is currently not supported for big bird
     test_head_masking = False
```

Comment:
head masking & pruning is not that easy for chunked attention and a very niche feature, so we'll disable it for now :-)